priv/repo/migrations/20170803180536_add_notification_configuration.exs

```diff
@@ -0,0 +1,17 @@
+defmodule CoursePlanner.Repo.Migrations.AddNotificationConfiguration do
+  use Ecto.Migration
+
+  def up do
+    alter table(:users) do
+      add :notified, :date
```

Comment:
please rename this field to `notified_at`, it is the convention to use `*_at` for the timestamp of an action, for example `inserted_at`, or `updated_at`